app/pom.xml

```diff
@@ -25,11 +25,30 @@
             <artifactId>quarkus-resteasy-jackson</artifactId>
         </dependency>
 
+        <!-- Persistence -->
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-jdbc-h2</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-jdbc-postgresql</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-hibernate-orm</artifactId>
+        </dependency>
+
         <!-- Third Party Libraries -->
         <dependency>
             <groupId>org.apache.avro</groupId>
             <artifactId>avro</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
```

Comment:
We need to check with Emmanuel et al whether this is OK to use.

Reply:
It looks like it should work: https://quarkusio.zulipchat.com/#narrow/stream/187030-users/topic/lombok

Reply:
Well, it did not cause any issues atm, if it will become problematic, I'll remove it. It generates the code during compilation, so should be safe. I just like the convenience:)

Reply:
I agree that is *should* be safe. :)  But we should just confirm that it's fine with the Quarkus team with respect to productization.  I'll check with @emmanuelbernard's team about it.  Let's leave it in for now and remove it only if we need to.